bin/configure-cli-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+/etc/init.d/zookeeper start
+cd /marathon
+screen -m -d -S testing -L sbt 'run --internal_store_backend mem --master local'
```

Comment:
You use the mem storage backend: Intention? To make more real world integration tests, I would omit that parameter and use the default (zk).